deps_server.ts

```diff
@@ -14,6 +14,7 @@ export {
 export type {
 	ApplicationCommandChoice,
 	ApplicationCommandOption,
+	MessageOptions,
```

Comment:
Since everything is ran using interaction would it be better for this to be `InteractionMessageOptions`. There's no important difference really

> https://doc.deno.land/https://code.harmony.rocks/3f591254deb572e3ffb4f2349d9a181ea49d3e33/mod.ts/~/InteractionMessageOptions